client/dist/index.html

```diff
@@ -6,6 +6,7 @@
     <title>Nearby Attractions</title>
 </head>
 <body>
+    <div ref='map'></div>
```

Comment:
I think you will need to find a way to incorporate this with your bundle.js instead of using the HTML page, which won't be accessible when we do the proxy server setup as far as I know